### PR TITLE
review request

### PR DESCRIPTION
**Pull request by femitj**

### CODE REVIEW

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            </nav>
+    </header>
+
+    <section id='welcome'>
+        <div class='container'>
+            <div id='welcome-box'>
+                <div class='welcome-text'>
+                    <h1>decide your <span>future</span> now!</h1>
+                    <button id="signup-btn"><a id='btn' href="views/resgister.html">Sign Up</a></button>
+                </div>
```
Comment (andela-gike) [inline UI/index.html]: Could you correct the spelling of your sign up url

File: UI/styles/admin.css [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    color: green;
+    background-color: inherit;
+    font-family: inherit; /* Important for vertical align on mobile phones */
+    margin: 0; /* Important for vertical align on mobile phones */
+  }
+  
+  /* Add a red background color to navbar links on hover */
+  .navbar a:hover, .dropdown:hover .dropbtn {
+   /* background-color: green;*/
+    font-weight: 600;
```
Comment (andela-gike) [inline UI/styles/admin.css]: Remove commented or unused code

File: UI/views/admin/create-office.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    <link rel="stylesheet" type="text/css" media="screen" href="../../styles/index.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="navigation__wrapper">
+                <div class='container'>
+                <div><a href="../../index.html" id="header-title">Politico</a></div>
+                <!--<div class='nav-link'><a href="./.html">Create Government office</a></div>-->
```
Comment (andela-gike) [inline UI/views/admin/create-office.html]: Remove commented or unused code

**Review by andela-gike — CHANGES_REQUESTED** [review]

File: UI/index.html (hunk 1 above)
Comment (femitj) [reply UI/index.html]: Done, changes would reflect in the next PR request... 
Thank you,

File: UI/styles/admin.css (hunk 2 above)
Comment (femitj) [reply UI/styles/admin.css]: Implemented feedback on this already, PR request will be sent soon
Thanks alot

**Comment by andela-gike** [comment]
You can close this PR it is no longer needed

**Pull request closed without merging**